jaguar-gpu/jaguar-gpu.c

```diff
@@ -3,208 +3,280 @@
  * Copyright (C) 2018 - Sebastien Alaiwan
```

Comment:
I think it's better to convert extras to SPDX headers too: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#licenses